packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.vue

```diff
@@ -357,9 +404,21 @@ export default {
 <style>
   .ProseMirror p.is-editor-empty:first-child::before {
     content: attr(data-placeholder);
-    float: left;
     color: var(--dt-color-foreground-placeholder);
     pointer-events: none;
     height: 0;
   }
+
+  .ProseMirror ul > li {
+    list-style-type: circle;
+  }
+
+  .ProseMirror ol > li {
+    list-style-type: decimal;
+  }
+
+  .ProseMirror blockquote {
+    padding-left: 1rem;
+    border-left: 3px solid var(--dt-color-foreground-muted-inverted);
+  }
```

Comment:
Correct border color and remove indenting.

```
  border-left: var(--dt-size-border-200) solid var(--dt-color-border-subtle);
  margin-left: 0;
```

Reply:
I'll add the left margin, but do we want this? or the style suggested in [comment above](https://github.com/dialpad/dialtone/pull/113#discussion_r1472916077)?

Reply:
Yeah. Zero out the blockquote is preferred rendering. The border color is also key, it's currently using a foreground Design Token rather than border Design Token.

<img width="357" alt="image" src="https://github.com/dialpad/dialtone/assets/1165933/8c7b875e-c657-41a0-af9c-0d77399e987c">